src/main/java/dc/Robot.java

```diff
@@ -55,6 +57,50 @@
   public static final double[] ENCODER_OFFSETS = {-Math.toRadians(67.852), -Math.toRadians(221.924), -Math.toRadians(214.980), -Math.toRadians(168.398)};
   public static final double STEER_RATIO = (15.0 / 32.0) * (10.0 / 60.0);
 
+  public static final boolean IS_COMPETITION = true;
+  private static final boolean USE_MODULES = true;
+  private static final int DRIVETRAIN_FRONT_LEFT_DRIVE_MOTOR = 1, DRIVETRAIN_FRONT_RIGHT_DRIVE_MOTOR = 4,
+          DRIVETRAIN_BACK_LEFT_DRIVE_MOTOR = 7, DRIVETRAIN_BACK_RIGHT_DRIVE_MOTOR = 10;
+  private static final int DRIVETRAIN_FRONT_LEFT_ANGLE_MOTOR = 2, DRIVETRAIN_FRONT_RIGHT_ANGLE_MOTOR = 5,
+          DRIVETRAIN_BACK_LEFT_ANGLE_MOTOR = 8, DRIVETRAIN_BACK_RIGHT_ANGLE_MOTOR = 11;
+  private static final int DRIVETRAIN_FRONT_LEFT_ENCODER_PORT = 3, DRIVETRAIN_FRONT_RIGHT_ENCODER_PORT = 6,
+          DRIVETRAIN_BACK_LEFT_ENCODER_PORT = 9, DRIVETRAIN_BACK_RIGHT_ENCODER_PORT = 12;
+  private static final double DRIVETRAIN_FRONT_LEFT_ENCODER_OFFSET = -Math.toRadians(67.852);
+  private static final double DRIVETRAIN_FRONT_RIGHT_ENCODER_OFFSET = -Math.toRadians(221.924);
+  private static final double DRIVETRAIN_BACK_LEFT_ENCODER_OFFSET = -Math.toRadians(214.980);
+  private static final double DRIVETRAIN_BACK_RIGHT_ENCODER_OFFSET = -Math.toRadians(168.398);
+
+  // Changes swerve modules & disables subsystems missing from the swerve test bot
```

Comment:
I assume this comment was copied from the Rapid React repo and that's why it mentions disabling subsystems. Might make sense to change to fit this repo so it doesn't cause any confusion in the future.